UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Views/NoticestoMariners/Leisure.cshtml

```diff
@@ -37,120 +33,20 @@
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5615SC5615_East_Coast_Whitby_to_Edinburgh.pdf" target="_blank">SC5615 East Coast - Whitby to Edinburgh</a></td>
-                        <td>2,482 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5615SC5615_East_Coast_Whitby_to_Edinburgh.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5616SC5616_Point_of_Ardnamurchan_to_Shiant_Islands.pdf" target="_blank">SC5616 Ardnamurchan to Shiant Islands</a></td>
-                        <td>1,631 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5616SC5616_Point_of_Ardnamurchan_to_Shiant_Islands.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5617SC5617_Fife_Ness_to_Inverness_and_the_Caledonian_Canal.pdf" target="_blank">SC5617 Fife Ness to Inverness and the Caledonian Canal</a></td>
-                        <td>1,209 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5617SC5617_Fife_Ness_to_Inverness_and_the_Caledonian_Canal.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5620SC5620_South_West_Wales.pdf" target="_blank">SC5620 South West Wales</a></td>
-                        <td>464 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5620SC5620_South_West_Wales.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5621SC5621_Carlingford_Lough_to_Waterford.pdf" target="_blank">SC5621 Carlingford Lough to Waterford</a></td>
-                        <td>1,944 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5621SC5621_Carlingford_Lough_to_Waterford.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5622SC5622_Ireland_-_South_Coast.pdf" target="_blank">SC5622 Ireland - South Coast</a></td>
-                        <td>1,066 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5622SC5622_Ireland_-_South_Coast.pdf" target="_blank">Download</a></td>
-                    </tr>
-                    <tr>
-                        <td><a href="~/nms/leisure/5623SC5623_Ireland_-_South_West_Coast.pdf" target="_blank">SC5623 Ireland - South West Coast</a></td>
-                        <td>622 KB (pdf)</td>
-                        <td><a href="~/nms/leisure/5623SC5623_Ireland_-_South_West_Coast.pdf" target="_blank">Download</a></td>
-                    </tr>
+                    @for (int i = 0; i < Model.Count; i++)
+                    {
+                        <tr>
+                            <td id="filename_@i">@Model[i].FileDescription</td>
```

Comment:
The Figma prototype has the file names linked for download too, as the static page did prior to this.

![image](https://user-images.githubusercontent.com/46884107/178415632-b6a5a11d-840e-4602-9a28-b9e27d53bbdf.png)

As suggested for cumulative page, this one also need to check with Brent whether we should stick with this design or not.

Reply:
Link added to file name as discussed in show and tell